client/components/AutoCompleteDepartment.js

```diff
@@ -1,15 +1,15 @@
 import React, { useMemo, useState } from 'react';
 import { AutoComplete, Option } from '@rocket.chat/fuselage';
 
-import { useTranslation } from '../contexts/TranslationContext';
 import { useEndpointData } from '../hooks/useEndpointData';
 
 export const AutoCompleteDepartment = React.memo((props) => {
-	const t = useTranslation();
 	const [filter, setFilter] = useState('');
 	const { value: data } = useEndpointData('livechat/department', useMemo(() => ({ text: filter }), [filter]));
 
-	const options = useMemo(() => (data && [{ value: 'all', label: t('All') }, ...data.departments.map((department) => ({ value: department._id, label: department.name }))]) || [{ value: 'all', label: t('All') }], [data, t]);
+	const { label } = props;
+
+	const options = useMemo(() => (data && [{ value: 'All', label: label && label }, ...data.departments.map((department) => ({ value: department._id, label: department.name }))]) || [{ value: 'All', label: label && label }], [data, label]);
```

Comment:
What happens when `label` is not defined?
Should not be `t('All')` a fallback when label is not defined?